hoomd/SnapshotSystemData.cc

```diff
@@ -39,6 +39,40 @@ void SnapshotSystemData<Real>::replicate(unsigned int nx, unsigned int ny, unsig
     pair_data.replicate(n, old_n);
     }
 
+template<class Real> void SnapshotSystemData<Real>::wrap()
+    {
+    vec3<double> frac;
+    vec3<double> wrapped;
+    vec3<double> pos;
+    vec3<double> fin;
+    int3 img;
+
+    for (unsigned int i = 0; i < particle_data.size; i++)
+        {
+        pos = particle_data.pos[i];
```

Comment:
Declare these variables inside the loop, as `auto const`.
```suggestion
        auto const pos = particle_data.pos[i];
```

Reply:
I think I've implemented this now